templates/frontend/objects/article_details.tpl

```diff
@@ -93,42 +93,49 @@
 				</p>
 			{/if}
 
-			{* Authors *}
-			{if $article->getAuthors()}
+			{* Authors & affiliations *}
+			{assign var="authors" value=$article->getAuthors()}
+			{assign var="hasAffiliations" value=0}
+
+			{foreach from=$authors item=author}
+				{if $author->getLocalizedAffiliation()}
+					{assign var="hasAffiliations" value=$hasAffiliations+1}
```

Comment:
I think we can just break the loop after this assignment, so avoid further iterations. See: https://www.smarty.net/docs/en/language.function.foreach.tpl#idp35399632. For such operations, I usually assign booleans - false and true (not numbers) but this doesn't have much importance.

Reply:
Thank you! Fixed in the last commit.